practice-01/js/index.js

```diff
@@ -68,11 +108,28 @@ function checkCharacter(input) {
         return false;
     }
 }
+
 /**
- * Validation for limit of number
+ * Display a message to the user when the content contains special characters
+ * @param isCharacterError : True || False
+ * @param input : Element of the argument
  */
-function checkLimitNumberError(input, min, max) {
+function validationFullName(isCharacterError, input) {
+    if (isCharacterError == false) {
+        showError(input, 'Name cannot contain characters')
+    } else {
+        showSuccess(input)
+    }
+}
 
+/**
+ * Check if the value of the number meets the requirements or not
+ * @param input : Element of the argument
+ * @param min : Minimum value that the user passed in
+ * @param max : Maximum value that the user passed in
+ * @returns {boolean}
+ */
+function checkLimitNumberError(input, min, max) {

```

Comment:
Please refactor this one.
Can we limited if/else?
Can we do `value < min` instead of `input.value < min`